gobblin-yarn/src/main/java/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -147,18 +141,8 @@ public GobblinApplicationMaster(String applicationName, Config config) throws Ex
 
     this.serviceManager = new ServiceManager(services);
 
-    List<Tag<?>> tags = ImmutableList.<Tag<?>>builder()
```

Comment:
We have to switch back to use `MetricRegistry` as `MetricContext` no longer allows registering standard Dropwizard metrics, like the ones registered in this class.